source/Dockerfile

```diff
@@ -17,5 +17,6 @@ WORKDIR /tmp
 ARG CONTROL_LIBRARIES_BRANCH=v7.0.0
 RUN git clone -b ${CONTROL_LIBRARIES_BRANCH} --depth 1 https://github.com/aica-technology/control-libraries.git
 RUN cd control-libraries/source && ./install.sh --auto --no-controllers --no-dynamical-systems --no-robot-model
+RUN cd control-libraries/protocol && ./install.sh --auto
```

Comment:
I don't see where this is currently used (although it had been previously discussed, we will likely tend away from clproto dependencies)